protocol/dubbo/pool.go

```diff
@@ -63,6 +74,8 @@ func newGettyRPCClientConn(pool *gettyRPCClientPool, protocol, addr string) (*ge
 			getty.WithReconnectInterval(pool.rpcClient.conf.ReconnectInterval),
 		),
 	}
+	gettyRPCClients.Store(addr, c)
+
```

Comment:
Load and Store maybe an atomic operating？ So I suggest a double `Load` check here.